src/order/galois_connection.lean

```diff
@@ -256,6 +276,16 @@ protected lemma dfun {ι : Type u} {α : ι → Type v} {β : ι → Type w}
 
 end constructions
 
+lemma l_comm_of_u_comm
```

Comment:
Can we also have the converse (ie `u_comm_of_l_comm`)?

Reply:
Sure (although I don't have any application in mind).